### ISSUE TITLE
Pan_genome file does not list cluster information

### ISSUE THREAD

**Issue by kachroo** [sanger-pathogens/Roary#242]
Hi,

I recently ran the Roary pipeline on some bacterial strains and the pan_genome_reference.fa file lists only the source sequence but not the cluster it came from. This is making it difficult for me to easily correlate the information in the gene_presence_absence.csv file and the pan_genome_reference.fa file. 
I used the following command to run Roary:

roary -p22 -e -n -qc -k /path/kraken -r -s -v *.gff 

Also, attached is the snippet from the two files. 
[pan_gen_ref.fa.pdf](https://github.com/sanger-pathogens/Roary/files/195938/pan_gen_ref.fa.pdf)
[gene_presence_absence.pdf](https://github.com/sanger-pathogens/Roary/files/195939/gene_presence_absence.pdf)

Thanks,
Priyanka

**Comment by andrewjpage** [comment]
Hi Priyanka, 
Which version of Roary are you running?  I added in the name of the cluster to the pan genome reference fasta 6 months ago?
Regards,
Andrew

**Comment by kachroo** [comment]
Hi Andrew,

Thanks for getting back to me. We had version 3.2.8 installed on our computer. Based on the version mentioned in the Roary instructions at http://sanger-pathogens.github.io/Roary/, we realized that there  maybe a newer version of the software available. Last week we download the version labeled 3.6.0 from github and although the tar.gz file says its v3.6.0, installation of the software and running raory -w lists v3.2.8. I generated the output based on our most recent Roary installation. 

I would appreciate your help in this regard.

Best,
Priyanka

**Comment by andrewjpage** [comment]
Hi Priyanka, 
It looks like the old version is still being picked up. The easiest way to install/upgrade it is from CPAN:

cpanm Bio::Roary

Andrew

**Comment by andrewjpage** [comment]
Theres no need to download any files manually. Just install it on the command line with cpan or cpanm, as you would with any Perl module.
 
    cpanm Bio::Roary

or

    cpan
    install Bio::Roary

This downloads and installs all the Perl dependancies, and sets the PERL5LIB correctly and should take less than a minute to get a working version.  Whilst by default it installs system wide, you can choose to install it into another directory (like your home dir) which doesnt require root access.

**Comment by kachroo** [comment]
Hi Andrew,

We purged to the best of our ability all prior installations of previous versions of Roary. We do not work with cpan very much, we are new to this. We then used cpan to install Bio::Roary as you suggested. The installation seemed to work fine, no error messages were thrown, all tests passed. When we run a "which roary" in the teminal we get: /home/sbberes/perl5/bin/roary.  If we open this file it indicates that it is version 3.6.0.  However if we invoke "roary -w" in the terminal we get: 3.2.8.
We reran roary on a small set of ten strains (i.e 10 .gff files) and the pan_genome_reference.fa file still does not list the cluster id associated with each representative strain. We do not understand why Roary still identifies itself as v3.2.8 instead of 3.6.0 and in output pan_genome_reference.fa file not updating the cluster Id information. Could this be due to multiple roary lib files/perl lib directories being resident at different locations on our computer at this point in time due to prior installations?

I have attached the terminal output generated when we ran the cpan install Bio::Roary. 
Any help you provide will be greatly appreciated. 

[roary.txt](https://github.com/sanger-pathogens/Roary/files/198150/roary.txt)
[roary-cpan-install.txt](https://github.com/sanger-pathogens/Roary/files/198151/roary-cpan-install.txt)


Thanks,
Priyanka
